### PR TITLE
Fixes Issue 35286 Firefox inconsistent scrolling

### LINKED ISSUE #35286: [Bug]: Firefox inconsistent scrolling
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Because Google Chrome is phasing out Manifest v2, I am moving back to Firefox, here I am seeing inconsistent scrolling in Nextcloud.

I have a folder with ~400 files and 4 folders, when I scroll down with the mouse wheel 3 clicks, the screen turns black, I see Nextcloud iterating through files in that status bar, next thing I see is I am at the bottom of the folder.

This happens regardless of sorting mode, and it works normally in Google Chrome.

### Steps to reproduce

1. Use Firefox 106.0.4 (64-bit) or Firefox developer edition 108.0b3 (64-bit)
2. Open folder, scroll down with mouse wheel, 3 clicks
3. Black screen, wait a while, and then we are at the bottom of the file list:

Here is a video:

https://user-images.githubusercontent.com/4353213/202892660-5d935a9c-a8d5-4698-b984-4a5196de0f67.mp4

### Expected behavior

Scroll down so I can see all the files in correct order without jumping down the list.

### Installation method

Community Manual installation with Archive

### Operating system

Other

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Enabled:
  - bruteforcesettings: 2.5.0
  - calendar: 4.1.0
  - cloud_federation_api: 1.8.0
  - contacts: 5.0.1
  - dav: 1.24.0
  - external: 5.0.0
  - federatedfilesharing: 1.15.0
  - files: 1.20.1
  - files_external: 1.17.0
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - oauth2: 1.13.0
  - passwords: 2022.11.20
  - provisioning_api: 1.15.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - text: 3.6.0
  - theming: 2.0.1
  - theming_customcss: 1.12.0
  - twofactor_backupcodes: 1.14.0
  - twofactor_totp: 7.0.0
  - user_saml: 5.0.3
  - viewer: 1.9.0
  - workflowengine: 2.7.0
Disabled:
  - activity: 2.5.2
  - admin_audit
  - circles: 22.1.1
  - comments: 1.11.0
  - contactsinteraction: 1.1.0
  - dashboard: 7.0.0
  - encryption
  - federation: 1.10.1
  - firstrunwizard: 2.1
  - nextcloud_announcements: 1.9.0
  - notifications: 2.1.2
  - password_policy: 1.3.0
  - photos: 1.0.0
  - privacy: 1.0.0
  - recommendations: 1.0.0
  - sharebymail: 1.10.0
  - support: 1.0.0
  - survey_client: 1.0.0
  - suspicious_login
  - systemtags: 1.2.0
  - updatenotification: 1.12.0
  - user_ldap: 1.15.0
  - user_status: 1.0.0
  - weather_status: 1.0.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by sorbaugh**
* Resolves: #35286

## Summary

URL: https://github.com/nextcloud/server/issues/35286

Adding overflow-anchor: none to #app-content element within files.scss and re-minified files.css and merged.css.

Tested on Firefox 111.0 (64-Bit)
## TODO

- [ ] ...

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/css/files.scss [hunk 1]
```diff
@@ -114,6 +114,7 @@
 	// force  the width to be the full width to not go bigger than the screen
 	// flex will grow for the mobile view if necessary
 	width: calc(100% - 300px);
+	overflow-anchor: none;
```
Comment (artonge) [inline apps/files/css/files.scss:117]: Can you add a comment explaining why this is needed ?

**Comment by artonge** [comment]
Can you also run `npm ci` and `npm run build`, and also amend your commit with a sign-off? Otherwise, CI will not pass.

**Comment by sorbaugh** [comment]
Hello! I ran npm run build as requested, which altered 98 files in the /dist directory of my fork. The purpose of these changes seem to be mainly for cache-busting / adding a new version number to the source-mapping url. 

Since I'm new to contributing in nextcloud and the main change is so small, it feels kind of overkill to commit this many files. So before I commit I wanted to ask if this is ok?

Best regards

**Comment by artonge** [comment]
> Since I'm new to contributing in nextcloud and the main change is so small, it feels kind of overkill to commit this many files. So before I commit I wanted to ask if this is ok?

Indeed, not the best, but this is how we currently do things :)

**Comment by sorbaugh** [comment]
Ok I assume the conflicts with the /dist directory were to be expected if npm run build is run frequently :)

If everything else is alright and ready for merge, I can do a rebase.

File: apps/files/css/files.scss (hunk 1 above)
Comment (artonge) [reply apps/files/css/files.scss:123]: :blue_heart:

**Pull request closed without merging**